src/QSymbolicsBase/predefined_fock.jl

```diff
@@ -161,4 +180,42 @@ julia> qsimplify(S*vac, rewriter=qsimplify_fock)
     basis::FockBasis
 end
 SqueezeOp(z::Number) = SqueezeOp(z, inf_fock_basis)
-symbollabel(x::SqueezeOp) = "S($(x.z))"
\ No newline at end of file
+symbollabel(x::SqueezeOp) = "S($(x.z))"
+
+"""Thermal bosonic state in defined Fock basis."""
+@withmetadata struct ThermalState <: AbstractSingleBosonOp
```

Comment:
Similar to the question about EPRPair name, this can also lead to confusion. There are thermal states of qubits and other things. Maybe `CVThermalState` or `BosonicThermalState` or something else that conveys that information. Probably makes sense to sync the name with the one for EPRPair.